### DISCUSSION TITLE
How can I create pull requests in laravel and avoid permission denied?

### DISCUSSION THREAD

**Discussion by ghost** [laravel/framework#50614] [Q&A]
I want to contribute with 2 MRs on laravel/framework and on laravel/laravel but when I push a feat or bug branch I get permission denied for my user. Each user needs to be manually granted for permission?

Or I have to fork the project first?

**Comment by Sophist-UK** [comment]
Yes - you need to fork the project and then clone the project to your PC.

Create a new branch in the clone, make your changes, commit and push to your fork on Github.

Then you can create a PR from this branch in your fork into the main branch of the Laravel framework repo.

**Reply by ghost** [reply]
Thanks.

**Comment by osama-98** [comment]
@Sophist-UK 
I have added a new PR to the laravel 11.x, and this PR is adding a new validation rules, which are **prohibited_if_declined** and **prohibited_if_accepted**

I added the PR to the 11.x branch, is this the correct branch, or should I add it to the master branch?

The pull-request: [https://github.com/laravel/framework/pull/54439](https://github.com/laravel/framework/pull/54439)

**Reply by Sophist-UK** [reply]
Now you wait and see if Taylor is interested or whether he will close it unmerged.

**Comment by ghost** [comment]
@osama-98 have a lok also at this pls https://github.com/laravel/framework/issues/51825
